src/create.rs

```diff
@@ -638,78 +602,12 @@ fn parse_create_function<'a>(
         }
     }
 
-    let mut characteristics = Vec::new();
```

Comment:
Where did the characteristic go?

Reply:
function characteristics  is not suitable for parse function body. function body is more like block statement. So I add Return sql in statement for parse function body as a block statement.